week-2/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,16 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+  if (mark >= 80) {
+    return "A";
+  } else if (mark < 80 && mark > 60) {
+    return "B";
+  } else if (mark <= 60 && mark > 50) {
```

Comment:
You don't need to check the `< 80` and `<= 60` conditions since they are already guaranteed by the previous branches of the statement. So if  `mark >= 80` the first branch is used. By definition, that means if the second branch is tested, we already know `mark < 80`.

You also don't need to use `else` here. Since each branch contains a `return` statement, the function will exit at that point, so you can write:

```
 if (mark >= 80) {
    return "A";
  }
  if (mark > 60) {
    return "B";
  }
  ...
```